views/courses/components/topics.hbs

```diff
@@ -62,12 +62,14 @@
 								</a>
 								<a href="/courses/{{../courseId}}/topics/{{../_id}}/edit"
 									class="btn btn-secondary btn-sm btn-icon-card"
+									aria-label="{{$t "global.button.editTopic"}}"
 									title="{{$t "global.button.editTopic"}}">
 									<i class="fa fa-pencil"></i>
 								</a>
 								{{/unless}}
 								<a href="#" data-href="{{../_id}}" data-courseId="{{../courseId}}"
 									class="btn btn-secondary btn-sm btn-create-share btn-icon-card"
+									aria-label="{{$t "courses._course.topic.text.shareTopic"}}"
 									title="{{$t "courses._course.topic.text.shareTopic"}}">
```

Comment:
Same

Reply:
I think aria-label or a title is not needed in places where there is a visual label in form of a span, you can remove them.

Reply:
Check for all such places in the code.